src/pages/Entries/Entries.tsx

```diff
@@ -15,26 +18,72 @@ export default function Entries() {
   const [editing, setEditing] = useState(false);
   const prevSelectedEntryId = useRef<number | undefined>(undefined);
   const { user } = useUser();
+  const { api } = useApi();
+  const queryClient = useQueryClient();
+  const numStillMutating = useIsMutating({ mutationKey: MutationKeys.MUTATE_HAPPINESS });
```

Comment:
just curious is this the mutation key thing you were talking about? (and just curious what does it do)

Reply:
Yeah this is part of it, this is just a hook to monitor the state of mutations. The key idea behind what I was describing earlier was the `invalidateQueries` function which allows us to use React query to tell other components that their data needs to be updated when data is changed. The way we know which queries need to be updated is with the query key.